src/blogContent/reactive-agents-khook.mdx

```diff
@@ -0,0 +1,52 @@
+}
+
+
+# Agent Cloud-Native in the House!
+
+Excited by the vision of smart AI agents watching over your Kubernetes cluster? 
+Want an easy fully cloud-native way to run your agentic software? 
+Time to discover [Kagent](https://kagent.dev)! 
+It’s a fairly young OSS project started by the folks at [Solo.io](https://Solo.io) which aims at making the building and running of AI Agents on Kubernetes easy and fun. With all the bells and whistles one would expect - authz and authn, security, visualization, governance and audit, optimization, you name it. Some of it on the roadmap, some of it being built as we speak.
+
+![Image description](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/d0tw6oe2omamz4u4x5ty.png)
+# Joining the Community
+I started playing with Kagent a couple of months ago, loved its ergonomics and decided to build a workshop around it - meanwhile fixing some docs, submitting PRs and joining the community meetings. If you’re looking for a great OSS community to join - look no further - it’s a warm and welcoming bunch of very smart folks. And there’s a lot of work to be done!
+
+# The Need for Reactivity
+So after working with Kagent for a while I realized something was missing. For me, anyway. You see - conceptually the difference between an agent and a tool is that an agent acts on your behalf, taking decisions in alignment with the declared goals and guidelines. (While with a tool - you need to take all of the decisions yourself and command your wishes.) In this respect - much of the cloud native software is already agentic. Anything built on KRM (
+[Kubernetes Resource Model](https://www.geeksforgeeks.org/linux-unix/kubernetes-resource-model-krm-and-how-to-make-use-of-yaml/)) is declarative by nature - the user declares the desired state and then the operators or the controllers make sure it becomes the actual state. And in that respect - they are definitely our agents - acting on our behalf, recreating missing pods, reconciling broken states. True agents are reactive by nature - they listen on events and correct course accordingly with state or goal declarations being just one type of event.
+
+And that’s exactly what Kagent didn’t have. Until now in order to summon an agent  - one needed to chat to it - either in Kagent’s sleek Web UI, over CLI or API.  But what good is an agent if it just sits there waiting for instructions? I wanted a way to make my agents reactive.
+
+# Khook
+
+![Image description](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/n8mzxcg00nd8jaybew8o.png)
+
+Enter Khook - a Kubernetes controller that allows defining:
+- Kubernetes events to listen to
+- the agent to call
+- the templated prompt to pass to the agent 
+
+Khook assumes autonomous remediation and incident response. Finally - the ops person's dream come true!
+
+The following diagram shows how Khook works:
+
+![Image description](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/74c60roop2xdyui3skl5.png) 
```

Comment:
I wonder if it would make more sense to have these images part of the repo/blog post, instead of hotlinking out to devto?

Reply:
Right, will do.